### PR TITLE
[fix](cache) fix show cache hotspot syntax error

### PR DESCRIPTION
**Pull request by dataroaring**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by dataroaring** [comment]
run buildall

**Review by airborne12 — DISMISSED** [review]
LGTM

**Review by morrySnow — DISMISSED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowCacheHotSpotStmt.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ private String generateQueryString() {
             query = q1.append(q2);
         } else if (metaDataPos == 2) {
             query = new StringBuilder("select partition_id as PartitionId, partition_name as PartitionName"
-            + "FROM " + TABLE_NAME.toString()
+            + " FROM " + TABLE_NAME.toString()
             + " where " +  whereExpr.get(0)
             + " and " + whereExpr.get(1)
             + "group by cluster_id, cluster_name, table_id, "
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowCacheHotSpotStmt.java:159]:
```suggestion
            + " group by cluster_id, cluster_name, table_id, "
```

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by airborne12 — APPROVED** [review]
LGTM

**Pull request merged**